packages/instant/src/components/zero_ex_instant.tsx

```diff
@@ -1,23 +1,75 @@
+import { AssetBuyer } from '@0x/asset-buyer';
+import { ObjectMap } from '@0x/types';
 import * as React from 'react';
 import { Provider } from 'react-redux';
 
+import { SelectedAssetThemeProvider } from '../containers/selected_asset_theme_provider';
 import { asyncData } from '../redux/async_data';
-import { store } from '../redux/store';
+import { INITIAL_STATE, State } from '../redux/reducer';
+import { store, Store } from '../redux/store';
 import { fonts } from '../style/fonts';
-import { theme, ThemeProvider } from '../style/theme';
+import { AssetMetaData, Network } from '../types';
+import { assetUtils } from '../util/asset';
+import { getProvider } from '../util/provider';
 
 import { ZeroExInstantContainer } from './zero_ex_instant_container';
 
 fonts.include();
-// tslint:disable-next-line:no-floating-promises
-asyncData.fetchAndDispatchToStore();
-
-export interface ZeroExInstantProps {}
-
-export const ZeroExInstant: React.StatelessComponent<ZeroExInstantProps> = () => (
-    <Provider store={store}>
-        <ThemeProvider theme={theme}>
-            <ZeroExInstantContainer />
-        </ThemeProvider>
-    </Provider>
-);
+
+export type ZeroExInstantProps = ZeroExInstantRequiredProps & Partial<ZeroExInstantOptionalProps>;
+
+export interface ZeroExInstantRequiredProps {
+    // TODO: Change API when we allow the selection of different assetDatas
+    assetData: string;
+    // TODO: Allow for a function that returns orders
+    liquiditySource: string;
```

Comment:
We spec'ed this out and it has to be called `liquiditySource` because it can also be a function. Open to changing this though. 
https://0xproject.quip.com/hBmlA07jXbEz/0x-Instant-Public-Spec

Reply:
Thanks for the link -- wasn't hip to this.  

The spec says `liquiditySource` can be "a function that returns a list of signed 0x orders".  If we are actually going to support that in the future, `liquiditySource`is a good name